### ISSUE TITLE
Kubernetes init error

### ISSUE THREAD

**Issue by anghelalexandra** [Morphl-AI/MorphL-Orchestrator#14]
Orchestrator install fails when the following command is run:

`kubeadm init --kubernetes-version=1.11.3 --pod-network-cidr=10.244.0.0/16`

Error log is:

```
[init] using Kubernetes version: v1.11.3
[preflight] running pre-flight checks
I0320 08:51:41.265077    7831 kernel_validator.go:81] Validating kernel version
I0320 08:51:41.265230    7831 kernel_validator.go:96] Validating kernel config
[preflight] The system verification failed. Printing the output from the verification:
KERNEL_VERSION: 4.15.0-1028-gcp
CONFIG_NAMESPACES: enabled
CONFIG_NET_NS: enabled
CONFIG_PID_NS: enabled
CONFIG_IPC_NS: enabled
CONFIG_UTS_NS: enabled
CONFIG_CGROUPS: enabled
CONFIG_CGROUP_CPUACCT: enabled
CONFIG_CGROUP_DEVICE: enabled
CONFIG_CGROUP_FREEZER: enabled
CONFIG_CGROUP_SCHED: enabled
CONFIG_CPUSETS: enabled
CONFIG_MEMCG: enabled
CONFIG_INET: enabled
CONFIG_EXT4_FS: enabled
CONFIG_PROC_FS: enabled
CONFIG_NETFILTER_XT_TARGET_REDIRECT: enabled (as module)
CONFIG_NETFILTER_XT_MATCH_COMMENT: enabled (as module)
CONFIG_OVERLAY_FS: enabled (as module)
CONFIG_AUFS_FS: enabled (as module)
CONFIG_BLK_DEV_DM: enabled
DOCKER_VERSION: 18.09.2
OS: Linux
CGROUPS_CPU: enabled
CGROUPS_CPUACCT: enabled
CGROUPS_CPUSET: enabled
CGROUPS_DEVICES: enabled
CGROUPS_FREEZER: enabled
CGROUPS_MEMORY: enabled
[preflight] Some fatal errors occurred:
	[ERROR SystemVerification]: unsupported docker version: 18.09.2
[preflight] If you know what you are doing, you can make a check non-fatal with `--ignore-preflight-errors=...`
```

**Comment by anghelalexandra** [comment]
Related issue: https://github.com/kubernetes/minikube/issues/3323

**Comment by anghelalexandra** [comment]
Closed by a3ea47d6c2f9fb1809e3e3100ce2769754bc8643
